lib/perl/Genome/Site/TGI/SiteLib/Genome/Disk/Group/Validate/GenomeDiskGroups.t

```diff
@@ -22,12 +24,12 @@ do {
         unix_uid => 0,
         unix_gid => 0,
     );
-    ok( ! exception { $DB::single = 1; $group->validate },
-        'no exception thrown when disk_group_name is GENOME_DISK_GROUP_MODELS'
-    );
+    ok(!$group->validate, 'no exception thrown when disk_group_name is GENOME_DISK_GROUP_MODELS');
```

Comment:
This could use `lives_ok`.  That way we don't have to care about the return value from `validate`.